samples/MortgageApplication/application-conf/file.properties

```diff
@@ -41,4 +41,13 @@ loadFileLevelProperties = true :: **/cobol/epsmlist.cbl
 # Example 1: Enable language configuration mapping for all COBOL files: loadLanguageConfigurationProperties=true :: **/cobol/*.cbl
 # Example 2: The following file path pattern looks for a language configuration properties file mapping for epsnbrvl.cbl and epsmpmt.cbl 
 # in the application-conf/languageconfigurationMapping.properties file:
-loadLanguageConfigurationProperties = true :: **/cobol/epsnbrvl.cbl, **/cobol/epsmpmt.cbl
\ No newline at end of file
+loadLanguageConfigurationProperties = true :: **/cobol/epsnbrvl.cbl, **/cobol/epsmpmt.cbl
+
+#
+# Properties mapping for assigning the language configuration to build files to specify the build configuration
+# Please see the detailed documentation on file property management at:
+# https://github.com/IBM/dbb-zappbuild/docs/FilePropertyManagement.md
+# This option replaces the mapping of build files to language configuration in languageConfigurationMapping.properties file
+# 
+languageConfiguration = languageConfigProps01 :: **/cobol/epsnbrvl.cbl, **/cobol/epsmpmt.cbl
+languageConfiguration = languageConfigProps02 ::  **/cobol/epscmort.cbl
```

Comment:
If line 44 only enables languageConfiguration for `**/cobol/epsnbrvl.cbl, **/cobol/epsmpmt.cbl `, then the languageConfiguration for `**/cobol/epscmort.cbl` will not be taken into account. Is that correct?

Reply:
Oh, this is correct, good spotted. Let me add this for `epscmort.cbl` . 

I noticed, that the test case does not pass for `epsnbrvl` either. I'll look into it.

Reply:
Ok - this led to update the expected values in the test framework.